src-electron/rest/user-data.js

```diff
@@ -20,744 +20,789 @@
  *
  * @module REST API: user data
  */
-const fs = require('fs')
```

Comment:
The whole file seem to have changed and makes it very difficult to actually review the changes.
Can you fix this?

Reply:
Fixed